nilearn/masking.py

```diff
@@ -635,19 +633,14 @@ def compute_brain_mask(target_img, threshold=.5, connected=True, opening=2,
     target_img = _utils.check_niimg(target_img)
 
     if mask_type == 'whole-brain':
-        template = load_mni152_template()
+        template = load_mni152_template(resolution=2)
```

Comment:
We need to pass the resolution provided to `compute_brain_mask` here (and same for the other two):

```python
template = load_mni152_template(resolution=resolution)
```

Reply:
Ups, my mistake. Fixed.